homeassistant/components/upnp/__init__.py

```diff
@@ -117,12 +118,13 @@ async def async_discover_and_construct(
             )
             _LOGGER.info("Detected multiple UPnP/IGD devices, using: %s", device_name)
 
-    ssdp_description = discovery_info["ssdp_description"]
-    return await Device.async_create_device(hass, ssdp_description)
+    ssdp_location = discovery_info[ssdp.ATTR_SSDP_LOCATION]
```

Comment:
Don't use the `ssdp` constant because the data is not sourced from the `ssdp` integration.